src/main/java/com/example/document/exceptionHandler/BenefitNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+package com.example.document.exceptionHandler;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.web.reactive.function.client.WebClientResponseException;
+
+import java.nio.charset.Charset;
+
+public class BenefitNotFoundException extends WebClientResponseException {
+    public BenefitNotFoundException(String message, int statusCode, String statusText, HttpHeaders headers, byte[] responseBody, Charset charset) {
+        super(message, statusCode, statusText, headers, responseBody, charset);
```

Comment:
Let's reduce amount of arguments for this method to make it more user-friendly.
For example we can pass an exception itself and extract most of the arguments from it and also Charset can be set here since we usually default to UTF_8